### ISSUE TITLE
EMRJobRunner has no _s3_fs attribute

### ISSUE THREAD

**Issue by conleyst** [Yelp/mrjob#2015]
We're trying to submit a job to a manually created EMR cluster, using `v0.6.7`. We get the following error,

```
Using configs in ./mrjob.conf
Traceback (most recent call last):
  File "mr_my_emr_job.py", line 235, in <module>
    MyEMRJob.run()
  File "<...>/venv/lib/python3.7/site-packages/mrjob/job.py", line 438, in run
    mr_job.execute()
  File "<...>/venv/lib/python3.7/site-packages/mrjob/job.py", line 465, in execute
    super(MRJob, self).execute()
  File "<...>/venv/lib/python3.7/site-packages/mrjob/launch.py", line 190, in execute
    self.run_job()
  File "<...>/venv/lib/python3.7/site-packages/mrjob/launch.py", line 235, in run_job
    runner.run()
  File "<...>/venv/lib/python3.7/site-packages/mrjob/runner.py", line 507, in run
    self._check_input_paths()
  File "<...>/venv/lib/python3.7/site-packages/mrjob/runner.py", line 1061, in _check_input_paths
    self._check_input_path(path)
  File "<...>/venv/lib/python3.7/site-packages/mrjob/emr.py", line 754, in _check_input_path
    for uri, obj in self._s3_fs._ls(path):
AttributeError: 'EMRJobRunner' object has no attribute '_s3_fs'
```
However, when we revert back to using `v0.6.6`, the job submits without a problem.

**Comment by davidmarin** [comment]
Yep, `_s3_fs` is part of mrjob's internal implementation and not guaranteed to be available across point releases.

In v0.6.7, try using `self.fs.s3._ls()` instead. :)

**Comment by stug** [comment]
@davidmarin I also ran into this today (essentially the same traceback), and I think I see the issue.

`EMRJobRunner._check_input_path()` (https://github.com/Yelp/mrjob/blob/v0.6.7/mrjob/emr.py#L744) references `self._s3_fs`, but `self._s3_fs` isn't set on the class until the `EMRJobRunner.fs` (https://github.com/Yelp/mrjob/blob/v0.6.7/mrjob/emr.py#L670) property has been referenced at least once.  I was able to fix this for a job by inserting a `job.fs` before `job.run()`.

**Comment by davidmarin** [comment]
Oh, wow, that's actually a pretty bad bug. The good news is, this has already been fixed in v0.6.8: `self._s3_fs` is now `self.fs.s3` (so `self.fs` will always get created first).

**Comment by davidmarin** [comment]
v0.6.8 should be out within the next week, FYI.
